### PR TITLE
Update wording

### PR DESCRIPTION
**Pull request by clochix**
- remove the price of Scaleway offer, as un user reported that Cozy may not be available on the 2.99 € offer;
 - update french translations from Transifex.

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -415,7 +415,7 @@
 	"offers description": "Try a pre-configured web server. But, be aware that it still requires technical skills to be properly maintained.",
 	"offers ovh name": "OVH - $3,49 / month",
 	"offers ovh text": "Low cost virtual Servers",
-	"offers scaleway name": "Scaleway - $3,49 / month",
```
Comment (frankrousseau) [inline src/locales/en.json]: The server is still available at that price.

File: src/locales/fr.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
 	"try main title": "Profitez d'un Cozy gratuit",
-	"try description": "Nous vous fournirons un serveur Cozy (hébergé en France) avec 60Go d'espace de stockage. En raison du grand nombre de demandes et du nombre de Cozy limité, il vous faudra attendre avant que nous puissions répondre à votre requête.",
+    "try description": "Nous vous fournirons un serveur Cozy (hébergé en France) avec 60 Go d'espace de stockage. En raison du grand nombre de demandes et du nombre limité de serveurs de test disponibles, il vous faudra probablement attendre un peu avant que nous puissions répondre à votre requête.",
```
Comment (frankrousseau) [inline src/locales/fr.json]: Indentation is wrong.

File: src/locales/fr.json [hunk 3]
```diff
@@ -321,8 +321,8 @@
 	"webdev features open": "Code open source",
 	"webdev features calendar": "Calendar, pour gérer votre emploi du temps",
 	"footer followus": "Suivez-nous",
-	"footer subscribe": "Inscrivez vous à nos news",
-	"footer subscribe button": "Go",
+    "footer subscribe": "Abonnez-vous à notre lettre d’information",
```
Comment (frankrousseau) [inline src/locales/fr.json]: Cette formulation est trop longue et va casser le style.

File: src/locales/fr.json [hunk 4]
```diff
@@ -321,8 +321,8 @@
...
+    "footer subscribe button": "M’abonner",
```
Comment (frankrousseau) [inline src/locales/fr.json]: Idem ici, il faut laisser un mot en deux lettres.

File: src/locales/fr.json [hunk 5]
```diff
@@ -410,13 +410,13 @@
...
-	"offers description": "Essayer un serveur pré-configuré. Toutefois, sachez que la maintenance requiert des compétences techniques.",
-	"offers ovh name": "OVH - 3,49 euros par mois",
-	"offers ovh text": "Serveurs virtuels pas chers",
-	"offers scaleway name": "Scaleway - 3,49 euros par mois",
-	"offers scaleway text": "Serveurs dédiés pas chers",
-	"offers sw name": "SWhosting - 45 euros par mois",
-	"offers sw text": "Serveurs virtuels haute disponibilité"
+    "apps go demo": "Voir la démo de",
+    "offers title": "ou commencez à utiliser Cozy dès à présent chez l’un de nos partenaires",
+    "offers description": "Essayez un serveur pré-configuré (et étant conscient que l’administration d’un tel serveur requiert quelques compétences techniques).",
```
Comment (frankrousseau) [inline src/locales/fr.json]: Pas fan de cette phrase qui sous-entend que les gens ont besoin de compétences techniques pour essayer.

**Comment by frankrousseau** [comment]
Thank you for the PR. I made comments.

File: src/locales/en.json (hunk 1 above)
Comment (clochix) [inline src/locales/en.json]: Yes, but a user reported that Cozy is not available on the 3.49 € offer. Cozy comes with a 150 Go disk, so the minimum price is 4.90 €. They may change so IMHO we shouldn't display this prices on our website.

File: src/locales/fr.json (hunk 2 above)
Comment (clochix) [inline src/locales/fr.json]: This line has been pulled from Transifex. Files pulled from Transifex seems to use space, whereas our current file uses tabs.

File: src/locales/fr.json (hunk 4 above)
Comment (clochix) [inline src/locales/fr.json]: I reverted both, but design issues shouldn't make us hurt the french language ☹

File: src/locales/fr.json (hunk 2 above)
Comment (frankrousseau) [inline src/locales/fr.json]: It's probably due to the metalsmith generator.

**Comment by clochix** [comment]
I updated the wording of the “Try it” page after discussing with Tristan.

**Pull request merged**